src/tsf/TfEditses.cpp

```diff
@@ -986,10 +986,10 @@ HRESULT CEditSessionUpdateCompositionString::_MakeInterimString(TfEditCookie ec,
         CConversionArea* conv_area = g_pConsoleTSF ? g_pConsoleTSF->GetConversionArea() : NULL;
         RETURN_HR_IF_NULL(E_FAIL, conv_area);
 
-        if (ResultStr) {
+        if (!ResultStr.empty()) {

```

Comment:
FWIW this particular `ResultStr` is always empty.

Reply:
Don't know why this exist, should I remove it?

Reply:
Yeah, pull it out. It's likely just a mistake. I went backwards in time to the older revisions of this code that weren't open sourced and they make the same mistake, so it doesn't look like an issue that the current generation of terminal/console team introduced.